authz/providers/azure/go.mod

```diff
@@ -0,0 +1,14 @@
+module github.com/krdhruva/guard/authz/providers/azure
```

Comment:
you wont need to change this if you keep appscode/guard :)

Reply:
Authz folder and code I added is not available in appscode so compilation was failing. Therefore I added to replace them in go.mod